frontend/package.json

```diff
@@ -14,6 +15,7 @@
     "husky": "^7.0.4",
     "leaflet": "^1.7.1",
     "lint-staged": "^12.3.7",
+    "lodash": "^4.17.21",
```

Comment:
I think that we don't need lodash, as it's an additional load for the project. Just try to refactor your function without using lodash

Reply:
Ok